subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -120,348 +111,16 @@ uint8_t ll_adv_aux_ad_data_set(uint8_t handle, uint8_t op, uint8_t frag_pref, ui
 		return BT_HCI_ERR_UNKNOWN_ADV_IDENTIFIER;
 	}
 
-	lll = &adv->lll;
-
-	/* Do not use Common Extended Advertising Header Format if not extended
-	 * advertising.
-	 */
-	pri_pdu_prev = lll_adv_data_peek(lll);
-	if (pri_pdu_prev->type != PDU_ADV_TYPE_EXT_IND) {
-		return ull_adv_data_set(adv, len, data);
-	}
```

Comment:
I may be missing something, but how is setting data on a legacy instance handled now?